velox/exec/SortBuffer.cpp

```diff
@@ -228,6 +232,34 @@ void SortBuffer::ensureInputFits(const VectorPtr& input) {
                << ", reservation: " << succinctBytes(pool()->reservedBytes());
 }
 
+void SortBuffer::ensureOutputFits() {
+  // Check if spilling is enabled or not.
+  if (spillConfig_ == nullptr) {
+    return;
+  }
+
+  // Test-only spill path.
+  if (testingTriggerSpill(pool_->name())) {
+    spill();
+    return;
+  }
+
+  const uint64_t outputBufferSizeToReserve =
```

Comment:
I think sort buffer has the estimated buffer size (estimatedOutputRowSize_) so we don't need the query config here.